cms/djangoapps/import_from_modulestore/api.py

```diff
@@ -0,0 +1,44 @@
+"""
+API for course to library import.
+"""
+from opaque_keys.edx.keys import LearningContextKey
+
+from .models import Import as _Import
+from .tasks import import_course_staged_content_to_library_task, save_legacy_content_to_staged_content_task
+from .validators import validate_usage_keys_to_import
+
+
+def create_import(source_key: LearningContextKey, user_id: int) -> _Import:
```

Comment:
```suggestion
def stage_content_for_import(source_key: LearningContextKey, user_id: int) -> _Import:
```
Let's make it clear that this kicks off the staging process in addition to creating the Import instance.

Reply:
Changed